app/pb_kits/playbook/pb_time_range_inline/_time_range_inline.scss

```diff
@@ -0,0 +1,27 @@
+[class^=pb_time_range_inline_kit] {
+  .align-right {
+    display: block;
+    margin-left: auto;
+    margin-right: 0;
+    width: max-content;
+  }
+  .align-center {
+    display: block;
+    margin: auto;
+    width: max-content;
+  }
+  .align-left {
+    display: block;
+    width: max-content;
+  }
```

Comment:
For these, you might be better off using the `Flex` kit to achieve the same/similar results.

Reply:
Using `Flex` now, thank you.